src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -1676,7 +1675,12 @@ public void canCreatePagedRequestWithNullProxyUser() {
   @Test
   public void requestCreationDoesNotFailWhenCirculationRulesReferenceInvalidNoticePolicyId() {
 
-    setInvalidNoticePolicyReferenceInRules("some-invalid-policy");
+    UUID invalidNoticePolicyId = UUID.randomUUID();
+    noticePoliciesFixture.create(new NoticePolicyBuilder()
+      .withId(invalidNoticePolicyId)
+      .withName("Example NoticePolicy"));
+    setInvalidNoticePolicyReferenceInRules(invalidNoticePolicyId.toString());
+    noticePoliciesFixture.cleanUp();
```

Comment:
Please only delete the create notice policy rather than all of them. I think this makes the situation being described in the scenario clearer.

Reply:
Sorry, missed it.
Updated

Reply:
Thanks